utils.py

```diff
@@ -76,3 +76,7 @@ def blockchain_transactions(node, wallet_address, limit, after=None):
             tx['direction'] = False
         txs_result.append(tx)
     return txs_result
+
+def amount_conversion_rate(amount, rate):
```

Comment:
maybe call this function 'apply_merchant_rate()'

Reply:
done